test/basic.test.js

```diff
@@ -106,39 +106,52 @@ tape('test invite.accept doesnt follow if already followed', function (t) {
         if (err) throw err
         // console.log(hops)
         t.equal(hops[bob.id], 1)
-        all(bob.messagesByType('pub'), function (err, ary) {
-          if (err) throw err
-          t.equal(ary.length, 1)
-
-          // it's rare, but some times, someone's home computer has a public address.
-          // this makes the tests fail unless we get the address the same way as invite code.
-          var expected = alice.address('public') || alice.address('local') || alice.address('device')
-
-          t.deepEqual({
-            type: 'pub',
-            address: ref.parseAddress(expected.split(';').shift())
-          }, ary[0].value.content)
 
-          all(bob.messagesByType('contact'), function (err, ary) {
+        bob.invite.accept(invite, (err) => {
+          t.error(err, 'no error')
```

Comment:
THIS is the only thing which _might_ be a breaking change.

It's no longer considered an error to accept and invite code that is valid